web/crux-ui/src/components/products/versions/images/edit-image-tags.tsx

```diff
@@ -6,14 +6,17 @@ import { TextFilter, textFilterFor, useFilters } from '@app/hooks/use-filters'
 import useTranslation from 'next-translate/useTranslation'
 import { useEffect, useState } from 'react'
 
-interface EditImageTagsProps {
+interface EditProps {
```

Comment:
Why the new name? It still selects an image tag imho

Reply:
It should be ImageTagInputProps so it can relate to the component

Reply:
Renamed the types so it matches the components.